examples/suit_update/README.md

```diff
@@ -483,10 +484,249 @@ success, the application reboots on the new slot:
+```
+
+This generates the manifest in SUIT CBOR format. The content can be inspected by
+using the `parse` subcommand:
+
+```console
+$ dist/tools/suit/suit-manifest-generator/bin/suit-tool parse -m suit_manifest
+```
+
+The manifest generated doesn't have an authentication wrapper, it is unsigned
+and will not pass inspection on the device or RIOT instance. The manifest can be
+signed with the `sign` subcommand together with the keys generated earlier.
+
+```console
+$ dist/tools/suit/suit-manifest-generator/bin/suit-tool sign -k keys/default.pem -m coaproot/suit_manifest -o coaproot/suit_manifest.sign
+```
+
+This generates an authentication to the manifest. This is visible when
+inspecting with the `parse` subcommand. The URL to this signed manifest will be
+submitted to the instance so it can retrieve it and in turn retrieve the
+component payload specified by the manifest.
+
+#### Updating the storage location
+
+The update process is a two stage process where first the instance receives the
+URL for the manifest. It will download the manifest and verify the content.
+After the manifest is verified, it will proceed with executing the command
+sequences in the manifest and download the payload when instructed to.
+
+The URL for the manifest can be submitted to the host with any coap client. In
+this example the `coap-client` from libcoap is used:
+
+```console
+$ coap-client coap://[2001:db8::2]/suit/trigger -m post -e 'coap://[2001:db8::1]/suit_manifest.signed'
+```
+
+The payload is the full URL to the signed manifest. The native instance should
+respond on this by downloading and executing the manifest. If all went well, the
+output of the native instance should look something like this:
+
+```
+suit: received URL: "coap://[2001:db8::1]/suit_manifest.signed"
+suit_coap: trigger received
+suit_coap: downloading "coap://[2001:db8::1]/suit_manifest.signed"
+suit_coap: got manifest with size 278
+suit: verifying manifest signature
+suit: validated manifest version
+)Retrieved sequence number: 0
+Manifest seq_no: 1, highest available: 0
+suit: validated sequence number
+)Formatted component name: .ram.0
+validating vendor ID
+Comparing 547d0d74-6d3a-5a92-9662-4881afd9407b to 547d0d74-6d3a-5a92-9662-4881afd9407b from manifest
+validating vendor ID: OK
+validating class id
+Comparing bcc90984-fe7d-562b-b4c9-a24f26a3a9cd to bcc90984-fe7d-562b-b4c9-a24f26a3a9cd from manifest
+validating class id: OK
+SUIT policy check OK.
+Formatted component name: .ram.0
+Fetching firmware |█████████████████████████| 100%
+Finalizing payload store
+Verifying image digest
+Starting digest verification against image
+Install correct payload
+Verifying image digest
+Starting digest verification against image
+Install correct payload
+```
+
+The storage location can now be inspected using the built-in command. If the
+same payload as suggested above was used, it should look like this:
+
+```Console
+> storage-content .ram.0 0 64
+41414242434344440A
+```
```

Comment:
I noticed that currently both ram.0 and ram.1 get written to, upon updating either ram.0 or ram.1
(at least that's what the output of the storage-content indicates on the console).
Maybe it's confusing ;)

Reply:
That sounds like a bug…

Reply:
And fixed in the latest push.